assets/dev/js/editor/controls/icons.js

```diff
@@ -0,0 +1,76 @@
+const ControlMultipleBaseItemView = require( 'elementor-controls/base-multiple' );
+class ControlIconsView extends ControlMultipleBaseItemView {
+	enqueueStylesheet( url ) {
+		if ( ! jQuery( document ).find( 'link[href="' + url + '"]' ).length ) {
```

Comment:
1. We already have `elementorCommon.elements.$document` in order to avoid multiple `jQuery` instances.
2. You should store your 'link' element as a class member, then access it anytime instead of searching across tens of thousands of DOM elements..
3. Avoid using of the same `jQuery` selector inside of one method.